### ISSUE TITLE
Completely Inaccessible with Screen Reader on Windows and MacOS

### ISSUE THREAD

**Issue by chigkim** [telegramdesktop/tdesktop#27234]
### Steps to reproduce

1. Run Telegram desktop
2. Run VoiceOver on MacOS with command+f5 (or triple click touch bar) or say "run voiceover" with Siri
3. Run Narrator on Windows with control+windows+enter
4. Try to navigate telegram window
5. VoiceOver: control+option+arrows
6. Narator: tab/shift+tab/arrows

### Expected behaviour

Screen reader should announce focused UI element as you navigate with keyboard.

### Actual behaviour

Nothing gets announced. QT has accessibility features for developers, and most stock QTWidget elements are accessible by default. Please see https://doc.qt.io/qt-6/accessible.html

### Operating system

Windows 10, MacOS 13

### Version of Telegram Desktop

4.12.2

### Installation source

Static binary from official website

**Comment by Aokromes** [comment]
duplicate.

**Comment by chigkim** [comment]
Could you let me know which one is duplicate? Thanks.
